docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
I'd favor keeping the plugins, keeping this documentation, just enhancing it to describe and maybe link more information about what these plugins do and why one would want to use them.

I was unaware of `gradle idea` before this PR. Ran it before my latest attempt to get a uPortal development environment going in IntelliJ-IDEA. It was significantly less miserable than a recent previous attempt. So, maybe a lot to be gained in guiding developers to run it before opening.